extensions/vertx-web-common/deployment/src/main/java/io/quarkus/vertx/web/common/deployment/AdditionalRoutesInstalledBuildItem.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.vertx.web.common.deployment;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Marker interface indicating the additional routes have been installed.
+ * <p>
+ * It allows to install the default route at the end.
+ */
+public final class AdditionalRoutesInstalledBuildItem extends MultiBuildItem {
```

Comment:
This is necessary to correctly phase the route declarations.

Reply:
I can see why, as we want to have them when we register the routes on the router.